CHANGES.md

```diff
@@ -1,3 +1,25 @@
+## edge-20.3.1
+
+This release introduces new functionality mainly focused around 
+observability and multicluster support. 
+
+* CLI
+  * Improved the `linkerd check` command to check for extension server
+    certificate (thanks @christyjacob4!)
+* Controller
+  * Removed restrictions preventing Linkerd from injecting proxies into
+    contour (thanks @alfatraining!)
+  * Added the SMI metrics API to the standard `linkerd install` flow. This
```

Comment:
nit: Looking at other "longer" bullets, semicolons are usually used instead of separate sentences. Maybe consider changing this to `.. 'linkerd install' flow; this is the first step ..`